gdal/frmts/netcdf/netcdfdataset.h

```diff
@@ -942,6 +953,8 @@ class netCDFDataset final: public GDALPamDataset
     netCDFDataset();
     virtual ~netCDFDataset();
     void SGCommitPendingTransaction();
+    void SGLogPendingTransaction();
+    std::shared_ptr<std::string> generateLogName();
```

Comment:
huh a shared_ptr<std::string> ? This is pretty unusual. Why not a straight std::string ?

Reply:
Will change it :+1: